### PR TITLE
New UI Design (step 2)

### PR DESCRIPTION
**Pull request by TG908**
This is step 2/4 of my big UI change. This PR aims to improve the model classes to simplify, the following steps (improving existing cards and creating new cards).

For some classes I have added some useful properties.
And some classes I made a collection of DataElements like for Grades and LibraryAccount (for displaying in one single cell)

I could use some help with the following things:

ToDo:
- [ ] Integrate the new Model classes (mainly: Grades, and Library Account) into the manager

### CODE REVIEW

File: TUM Campus App/Grades.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+//  Created by Florian Gareis on 04.02.17.
+//  Copyright © 2017 LS1 TUM. All rights reserved.
+//
+
+import Foundation
+
+
+class Grades: DataElement {
+    
+    let name: String
```
Comment (mathiasquintero) [inline TUM Campus App/Grades.swift]: I like the new Model Classes. These will actually go very well with the new refactored network layer. So I would probably put this off until we're ready to have both at the same time

File: TUM Campus App/Tuition.swift [hunk 2]
```diff
@@ -12,11 +12,16 @@ class Tuition: DataElement {
     let frist: Date
     let semester: String
     let soll: String
+    let amountDue: Double
+    var payed: Bool {
+        return amountDue == 0
+    }
     
-    init(frist:Date,semester:String,soll:String) {
+    init(frist:Date,semester:String,soll:String, amountDue: Double) {
```
Comment (mathiasquintero) [inline TUM Campus App/Tuition.swift]: If we're adding the `amountDue` I'd remove `soll` altogether.

File: TUM Campus App/Grades.swift [hunk 3]
```diff
@@ -0,0 +1,52 @@
+//
```
[the remark this answers is not in the document]
Comment (TG908) [reply TUM Campus App/Grades.swift]: 🙊

File: TUM Campus App/Grades.swift (hunk 1 above)
Comment (TG908) [reply TUM Campus App/Grades.swift]: Sounds great.

**Comment by TG908** [comment]
Implemented in #167 in parts

**Pull request closed without merging**